src/main.cpp

```diff
@@ -2528,13 +2528,13 @@ CConsensusFlags GetConsensusFlags(const CBlock& block, const CBlockIndex *pindex
 
     // Start enforcing the DERSIG (BIP66) rules, for block.nVersion=3 blocks,
     // when 75% of the network has upgraded:
-    if (block.nVersion >= 3 && IsSuperMajority(3, pindex->pprev, chainparams.GetConsensus().nMajorityEnforceBlockUpgrade, chainparams.GetConsensus())) {
+    if (pindex->nVersion >= 3 && IsSuperMajority(3, pindex->pprev, chainparams.GetConsensus().nMajorityEnforceBlockUpgrade, chainparams.GetConsensus())) {
```

Comment:
no, where it is used, pindex points to the block. (I ran the tests after this change, worked fine)